package.json

```diff
@@ -1,4 +1,25 @@
 {
+  "name": "pwr2",
+  "description": "Docker image based on Ubuntu providing a base setup for a Phoenix+Webpack+Relay+React project, with some sugar and conventions to develop and build your own web applications.",
+  "main": "webpack.config.js",  
+  "version": "",
+  "directories": {
+    "test": "test"
+  },
+  "keywords": [],
+  "preferGlobal": true,
+  "contributors": [
+    {
+      "name": "Javier Rios Selent",
```

Comment:
Is this "contributors" field required? 
Your PR will probably fix the issue #17 but I think we have to find a way to do one of the following: put here all the contributors or put something like *iporaitech*
What do you think @hisa?